### PR TITLE
WRO-8722: Update in-editor linting guide

### PR DESCRIPTION
**Pull request by hong6316**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Enact CLI supports in-editor linting by installing the 'eslint-config-enact-proxy' module locally.
But for ease to use, we need to guide users more kindly and in detail.

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
- Add a guide for installing `eslint-config-enact-proxy`

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)
- Change `remove` to `uninstall` when deleting globally-installed packages
- Update .travis.yml to solve the travis error
    - ubuntu 20 is required for node 18 ( https://docs.travis-ci.com/user/reference/focal/)

### Links
[//]: # (Related issues, references)
WRO-8722

### Comments
Enact-DCO-1.0-Signed-off-by: Taeyoung Hong(taeyoung.hong@lge.com)

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -52,16 +52,21 @@ You would need to install an ESLint plugin for your editor first.
...
+
+```sh
+npm install eslint-config-enact-proxy
+```
+
 In order for in-editor linting to work with our updated ESLint config, you'll need to upgrade to ESLint 7 or later. This can be installed globally by running:
 
 ```sh
-npm install -g eslint
+npm install --location=global eslint @enact/cli
```
Comment (MikyungKim) [inline docs/index.md:63]: This change seems not necessary.

File: docs/index.md [hunk 2]
```diff
@@ -52,16 +52,21 @@ You would need to install an ESLint plugin for your editor first.
 Ever since ESLint 6, global installs of ESLint configs are no longer supported.
 To work around this new limitation, while still supporting in-editor linting, we've created a new [eslint-config-enact-proxy](https://github.com/enactjs/eslint-config-enact-proxy) package.
 The [eslint-config-enact-proxy](https://github.com/enactjs/eslint-config-enact-proxy) acts like a small proxy config, redirecting ESLint to use a globally-installed Enact ESLint config.
+
+```sh
+npm install eslint-config-enact-proxy
```
Comment (MikyungKim) [inline docs/index.md:57]: Why did you put this here?

File: docs/index.md [hunk 3]
```diff
@@ -52,16 +52,21 @@ You would need to install an ESLint plugin for your editor first.
...
 ```sh
-npm install -g eslint
+npm install --location=global eslint @enact/cli
 ```
 
 Then, you will need to uninstall any previous globally-installed Enact linting package (everything but eslint itself):
 
 ```sh
-npm remove -g eslint-plugin-react eslint-plugin-react-hooks eslint-plugin-babel @babel/eslint-parser eslint-plugin-jest eslint-plugin-enact eslint-config-enact
+npm remove --location=global eslint-plugin-react eslint-plugin-react-hooks eslint-plugin-babel @babel/eslint-parser eslint-plugin-jest eslint-plugin-enact eslint-config-enact
```
Comment (MikyungKim) [inline docs/index.md:69]: Could you please add some references to changing `-g` option?
Also, `npm uninstall` seems proper here.

**Comment by MikyungKim** [comment]
I don't think the guide is complete. Please re-write it clearer.

**Review by MikyungKim — CHANGES_REQUESTED** [review]
I don't think the guide is complete. Please re-write it clearer.

File: docs/index.md (hunk 1 above)
Comment (hong6316) [reply docs/index.md:63]: I will revert this change

File: docs/index.md (hunk 2 above)
Comment (hong6316) [reply docs/index.md:57]: I will revert this change

File: docs/index.md [hunk 4]
```diff
@@ -106,6 +111,15 @@ Unfortunately, Syntastic does not support real-time linting and only lints on sa
 
 Install the [vscode-eslint](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint) plugin.  More information on linters and Visual Studio Code can be found [here](https://code.visualstudio.com/docs/languages/javascript#_linters).
 
+> **NOTE**: To open the Settings editor, use the following VS Code menu command:
```
[the remark this answers is not in the document]
Comment (hong6316) [reply docs/index.md:114]: I think these comments are not needed because the upper link includes these comments.

File: docs/index.md (hunk 3 above)
Comment (hong6316) [reply docs/index.md]: OK

File: docs/index.md (hunk 2 above)
Comment (hong6316) [reply docs/index.md]: I think some developers may use this example as a usage guide of `eslint-config-enact-proxy`.
Installing `eslint-config-enact-proxy` is also an important step but there's no guidance for installing it.

File: docs/index.md [hunk 5]
```diff
@@ -52,6 +44,12 @@ You would need to install an ESLint plugin for your editor first.
 Ever since ESLint 6, global installs of ESLint configs are no longer supported.
 To work around this new limitation, while still supporting in-editor linting, we've created a new [eslint-config-enact-proxy](https://github.com/enactjs/eslint-config-enact-proxy) package.
 The [eslint-config-enact-proxy](https://github.com/enactjs/eslint-config-enact-proxy) acts like a small proxy config, redirecting ESLint to use a globally-installed Enact ESLint config.
+In order for in-editor linting, `eslint-config-enact-proxy` should be installed locally on a project:
```
Comment (MikyungKim) [inline docs/index.md:47]:
```suggestion
`eslint-config-enact-proxy` needs to be installed locally on a project to enable in-editor linting:
```

File: docs/index.md [hunk 6]
```diff
@@ -52,6 +44,12 @@ You would need to install an ESLint plugin for your editor first.
...
+
+```sh
+npm install --save-dev eslint-config-enact-proxy
+```
+
```
Comment (MikyungKim) [inline docs/index.md:52]: Please guide to add `eslintConfig` property in package.json.

File: docs/index.md [hunk 7]
```diff
@@ -15,28 +15,20 @@ If you use the `cli` tools to create your project, `npm run lint` will run the E
 If you want to switch to the strict version of the linting rules, modify your `package.json` file and change the following lines:
 
 ```json
-    "lint": "enact lint",
-...
-  "eslintConfig": {
-    "extends": "enact-proxy"
-  },
+    "lint": "enact lint"
```
Comment (MikyungKim) [inline docs/index.md:18]: Let's not modify this.

File: docs/index.md (hunk 6 above)
Comment (hong6316) [reply docs/index.md:52]: OK

File: docs/index.md (hunk 5 above)
Comment (hong6316) [reply docs/index.md:47]: OK

**Review by MikyungKim — APPROVED** [review]
LGTM

**Pull request merged**